src/foundry/client/pixi/core/containers/base-canvas-group.d.mts

```diff
@@ -53,12 +65,16 @@ declare global {
    * @param ContainerClass - The parent Container class being mixed.
    * @returns A ContainerClass subclass mixed with BaseCanvasMixin features.
    */
-  function CanvasGroupMixin<BaseClass extends BaseCanvasMixin.BaseClass>(
+  function CanvasGroupMixin<BaseClass extends CanvasGroupMixin.BaseClass>(
     ContainerClass: BaseClass,
   ): Mixin<typeof CanvasGroup, BaseClass>;
 
-  namespace BaseCanvasMixin {
+  namespace CanvasGroupMixin {
     type BaseClass = typeof AnyPIXIContainer;
+
+    type DrawOptions = InexactPartial<Record<string, unknown>>;
+
+    type TearDownOptions = InexactPartial<Record<string, unknown>>;
```

Comment:
The `InexactPartial` is always redundant for a `Record<string, unknown>` or more generally a `Record<string, T | undefined>` (in this case `T = unknown`)

As in both of these work fine:
```ts
const a = Math.random() > 0.5 ? { foo: "foo" } : { bar: 1 }; // typed with optional properties.
const x: InexactPartial<Record<string, unknown>> = a;
const y: Record<string, unknown> = a;
```

I'd also recommend `AnyObject` in this case since that allows readonly properties.

I would also maybe instead change this to `Record<string, never>` because currently there's no properties. There's no perfect answer though because that is technically unnecessarily conservative and could mess with subclassing.